synthesis/codon/codon_test.go

```diff
@@ -1,22 +1,20 @@
 package codon
```

Comment:
The logic of all existing tests is kept the same (and they pass) but I should probably add some more. Let me know if you want a particular one!